azerothcore/azerothcore-wotlk
# fix(DB/Loot): Sets drop rate of Atal'ai items in accordance of the addressed issue

## Changes Proposed:
-  Sets drop rate of Atal'ai items in accordance of the addressed issue

## Issues Addressed:

- Closes #7895 
- Closes https://github.com/chromiecraft/chromiecraft/issues/1729

## SOURCE:


## Tests Performed:

- Tested as described below


## How to Test the Changes:


1. Run SQL query, both should change 6 rows
2. Verify drop rate of the items, see addressed AC issue
3. Verify chance of drop, ``SELECT * FROM `creature_loot_template` WHERE `Reference` = 35010;``, should all be 36 (Unless this reference table is added to another mob in the meantime)

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
